service-runner/src/main/resources/config/assets.yaml

```diff
@@ -14,6 +14,36 @@ assets:
       fee_fixed: 0
       fee_percent: 0
       max_amount: 1000000
+    sep6:
+      deposit:
+        exchange_enabled: true
+        fields:
+          email_address:
+            description: email address for transaction status updates
+            optional: true
+      withdraw:
+        exchange_enabled: true
+        types:
+          bank_account:
+            fields:
+              dest:
+                description: your bank account number
+              dest_extra:
+                description: your routing number
+              bank_branch:
+                description: address of your bank branch
+              phone_number:
+                description: your phone number in case there's an issue
+              country_code:
+                description: ISO-3166-1 alpha-3 code of the user's current address
+                choices:
+                  - USA
+                  - PRI
+          cash:
+            fields:
+              dest:
+                description: your email address. Your cashout PIN will be sent here.
+                optional: true
```

Comment:
❗ Same thing here -- lets remove this and assume its all configured and handled via SEP-12

Reply:
I think you mean only removing the KYC fields here, right? We would need to keep the financial account information (`type`, `dest`, `dest_extra`). SEP-9 does not currently support having multiple types of financial accounts.

Reply:
Discussed this with @JakeUrban @lijamie98. We will delegate the fields to SEP-12 and ignore the `dest` and `dest_extra` parameters for now.